### ISSUE TITLE
toolbars_below_content_v2.css breaks image themes (e.g. Firefox Alpenglow)

### ISSUE THREAD

**Issue by gotoAndDie** [MrOtherGuy/firefox-csshacks#495]
- Downloaded raw file from [94f5ef9](https://github.com/MrOtherGuy/firefox-csshacks/commit/94f5ef943f25270241e4781a2262d4d79376d80e)
- Firefox 133.0.3 (aarch64) for Fedora 40 (Asahi Linux)

Normally, I use a [Spirited Away](https://addons.mozilla.org/en-GB/firefox/addon/spirited-away-animated/) theme, which uses a fixed-size image to create its colour scheme. With the toolbars offset, the images are no longer shown, and only a grey background remains.

This issue can also be observed on the included (I think?) Firefox Alpenglow theme, which was supposed to have a bunch of glowy effects (which are presumably images), which are missing when toolbars_below_content_v2.css is applied.

Both themes work with toolbars_below_content.css, both before and after Firefox 133 - though the old version has different (expected ) issues on versions 133 and above.

**Comment by MrOtherGuy** [comment]
Yeah, I know about this. It should be pretty simple to patch this to work and I can do it tomorrow probably.

**Comment by MrOtherGuy** [comment]
This should be fixed with the latest patch, and hopefully without causing issues for non-linux systems.
